src/main/java/com/exasol/dbbuilder/dialects/exasol/ExasolImmediateDatabaseObjectWriter.java

```diff
@@ -122,6 +149,15 @@ public void write(final VirtualSchema virtualSchema) {
         writeToObject(virtualSchema, builder.toString());
     }
 
+    /**
+     * Drop a Virtual Schema.
+     *
+     * @param virtualSchema Virtual Schema to drop.
+     */
+    public void drop(final VirtualSchema virtualSchema) {
+        writeToObject(virtualSchema, "DROP VIRTUAL SCHEMA " + virtualSchema.getFullyQualifiedName() + " CASCADE");
```

Comment:
Why did you implement cascade only for virtual schemas? I think other Exasol drop commands should also be able to use cascade and it should be also decided by the user which one he prefers to use. And now it's hidden in the implementation

Reply:
As far as i found out, DROP VIRTUAL SCHEMA always requires the CASCADE. Therefore I added it.

Reply:
Are you 100% sure? I also always use CASCADE for virtual schemas, but according to the documentation on the portal, it's an optional value and an example also doesn't have it: https://docs.exasol.com/sql/drop_schema.htm

Please, check again. If it's possible to use drop without cascade with virtual schema, I suggest having a usual drop and drop with the cascade (same for other structures)